docs/epayment.v1.yml

```
@@ -749,10 +773,15 @@ components:
         timestamp:
           type: string
           format: date-time
+          example: 2022-12-31T00:00:00Z
         idempotencyKey:
           type: string
-          description: The Idempotency key of the request.
+          description: |-
+            The idempotency key of the request.
+            Specified by the merchant/partner making the API request.
           nullable: true
+          maxLength: 36
```

Updated after checking the code: https://github.com/vippsas/epayments-platform/blob/1f081f2ceb93b8e6a3867c80d2bf52a5f200819a/src/030_Presentation/010_Apis/ePayment.Api/Validation/ePayment/IdempotencyKeyValidationExtensions.cs#L29
```suggestion
          maxLength: 50
```